### DISCUSSION TITLE
Support for Pydantic v2

### DISCUSSION THREAD

**Discussion by farzbood** [RobertCraigie/prisma-client-py#797] [Polls]
Hi everyone,
Since FastAPI (0.100.0) released with support for Pydantic v2, which brings big improvments in app's performance, how about a version bump in **Prisma Client Python** to be compatible with Pydantic v2 specifications?
This is to call for this upgrade/bump to support FastAPI 0.100.0 and Pydantic v2.
Happy development, PEACE V ;)

**Comment by marcfalk** [comment]
For others stopping by this is being worked on here: https://github.com/RobertCraigie/prisma-client-py/pull/763
